lib/config/utils.ts

```diff
@@ -19,7 +19,8 @@ export function mergeChildConfig<
     if (
       option.mergeable &&
       childConfig[option.name] &&
-      parentConfig[option.name]
+      parentConfig[option.name] &&
+      parentConfig[option.name] !== childConfig[option.name]
```

Comment:
why this change? 🤔

Reply:
Without this I had the problem that my newName, newValue and finally depName changed from one string to an array of single characters.